### PR TITLE
Renaming css modules to .m.css

### PR DESCRIPTION
**Pull request by rorticus**
**Type:** bug / feature

The following has been addressed in the PR:

* [x] There is a related issue
* [x] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
* [ ] Unit or Functional tests are included in the PR

**Description:**

Renaming CSS modules to `.m.css`.  I tested this locally like this:

```shell
$ dojo create --name=bob
$ cd bob
$ dojo build
$ dojo test -a
# check built files to ensure visually ensure css
$ open http://localhost:8080
```

Resolves https://github.com/dojo/widgets/issues/76

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -36,16 +38,15 @@
...
     "glob": "^7.0.3",
     "grunt": "~1.0.1",
-    "grunt-dojo2": ">=2.0.0-beta.36",
+    "grunt-dojo2": ">=2.0.0-beta.39",
     "intern": "^3.4.1",
     "istanbul": "^0.4.3",
     "mockery": "^1.7.0",
     "remap-istanbul": "^0.6.4",
     "sinon": "^1.17.5",
+    "tslint": "^4.0.0",
```
Comment (agubler) [inline package.json]: shouldn't need this

**Review by agubler — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (rorticus) [reply package.json]: Without this, I was getting `Linter.Linter is not a constructor`

Comment (agubler) [reply package.json]: it should be provided by grunt-dojo2, wipe your node_modules and re-install

Comment (agubler) [reply package.json]: runs on master for me

Comment (rorticus) [reply package.json]: OK. Removing `node_modules` did the trick. @agubler how did you get to be so smart?

**Review by agubler — APPROVED** [review]

**Pull request merged**